dusty/systems/docker/__init__.py

```diff
@@ -26,6 +28,39 @@ def get_dusty_images():
 def get_dusty_container_name(service_name):
     return 'dusty_{}_1'.format(service_name)
 
+def registry_from_image(image_name):
+    """Returns the Docker registry host associated with
+    a given image name."""
+    if '/' not in image_name: # official image
```

Comment:
Uuuuuugh apparently there's many different ways to specify these but the other ones, e.g. `library/postgres`, should all hit the second return and still be correct